dsl/src/main/scala/org/http4s/dsl/impl/ResponseGenerator.scala

```diff
@@ -60,6 +60,28 @@ trait LocationResponseGenerator[F[_]] extends Any with EntityResponseGenerator[F
   def apply(location: Uri)(implicit F: Applicative[F]): F[Response[F]] =
     F.pure(
       Response[F](status = status, headers = Headers(`Content-Length`.zero, Location(location))))
+
+  /** A 300, 301, 302, 303, 307 and 308 status SHOULD contain a Location header, which
```

Comment:
Maybe this scaladoc works better at the trait level? This way we avoid the duplication.

Reply:
Fair enough, I was following the pattern in WwwAuthenticateResponseGenerator but happy to move doc to trait level.